### DISCUSSION TITLE
Using Admonitions inside content of Tabs

### DISCUSSION THREAD

**Discussion by seeergiofdez** [facebook/docusaurus#8430] [General] [answered]
Hello, I'm not sure if this question has already been answered, but I have this doubt: Can we **use admonitions inside the content of a tab**?

On the docusaurus documentation I found [Admonitions with MDX](https://docusaurus.io/docs/markdown-features/admonitions#admonitions-with-mdx), but what I want is to use a admonition inside the content of the tab and not the other way around. 

I tried:
```
<Tabs>
  <TabItem value="test1" label="Test 1" default>

  Content 1

  </TabItem>
  <TabItem value="test2" label="Test 2">

  Content 2

  :::info
  This is a info message
  :::
  
  </TabItem>
</Tabs>
```

But I got:
![tst](https://user-images.githubusercontent.com/45104843/207092522-e4ceb8e2-54ac-4d58-b585-d6f23185a66e.png)

Is it possible?
Thanks!

**Comment by slorber** [comment, marked as answer]
I'm not sure it's possible today without using MDX:


```mdx
:::tip Use tabs in admonitions

<Tabs>
  <TabItem value="apple" label="Apple">
     <Admonition type="info">
        <p>Some information</p>
      </Admonition>
  </TabItem>
  <TabItem value="orange" label="Orange">This is an orange 🍊</TabItem>
  <TabItem value="banana" label="Banana">This is a banana 🍌</TabItem>
</Tabs>

:::
```

![CleanShot 2022-12-14 at 15 31 38](https://user-images.githubusercontent.com/749374/207622943-692f9cb1-1d61-4e44-9751-a1403c0e38df.png)


However I think it could become possible after upgrading to MDX 2 (https://github.com/facebook/docusaurus/pull/8288)

**Reply by slorber** [reply]
Can confirm the following should work with MDX 2

```mdx
::::tip Use tabs in admonitions

:::info Use admonition nested

test

:::

<Tabs>
  <TabItem value="apple" label="Apple">

:::note Use admonition in tab

test

:::

  </TabItem>
  <TabItem value="orange" label="Orange">This is an orange 🍊</TabItem>
  <TabItem value="banana" label="Banana">This is a banana 🍌</TabItem>
</Tabs>

::::
```

![CleanShot 2022-12-15 at 13 49 23](https://user-images.githubusercontent.com/749374/207863261-765d07cf-68f0-4fa0-b3fc-152eb80596c7.png)

**Reply by seeergiofdez** [reply]
Hey! Thanks for the answer.
How can I use MDX 2 on Docusaurus? I don't find it clear on the pull request

**Reply by slorber** [reply]
The MDX 2 PR isn't merged yet. It will normally be in Docusaurus 3.0

**Reply by seeergiofdez** [reply]
Oh okay. So Amonitions inside Tabs is a feature that can be used in the next Docusaurus version. Is there any ETA for Docusaurus 3? Thanks

**Reply by slorber** [reply]
Yes

No ETA, when things are ready and it's hard to estimate.

When the PR above is merged we'll likely release an alpha/beta or something

**Reply by tanberry** [reply]
Thank you @slorber this helped me a lot!
